### DISCUSSION TITLE
blog/gradient-border-is-hard

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#414] [General]
# blog/gradient-border-is-hard

A simple gradient border is not as simple as it seems.

https://theodorusclarence.com/blog/gradient-border-is-hard

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Comment by PedroTrudes** [comment]
i don´t have asks, but I want to thank you for the content. Congratulations  🚀🚀 

My new favorite website!

**Reply by theodorusclarence** [reply]
Thanks!

**Comment by Rillwz** [comment]
I don't get it, how could you create this incredible sites, Highly Polished. 🤘

**Comment by m-shum** [comment]
Great article breaking this stuff down! Though I'm curious why this solution doesn't work without the blur filter.

**Reply by theodorusclarence** [reply]
Thanks! Can you elaborate?

**Comment by wemwi** [comment]
thank you so much!!!
just spend several hours till i found your post ❤️

**Comment by oreoluwa-bs** [comment]
Thank you very much!
